### PR TITLE
Supplemental FDA information

### PR DESCRIPTION
**Pull request by RickHawesUSDS**
This PR adds a supplemental information about the FDA authorized uses of tests. The new attributes that are calculated are: `test_authorized_for_otc`, `test_authorized_for_home`, and `test_authorized_for_unproctored`. These values come from the FDA authorization letters. 

## Changes
- A new LIVD-supplemental table that contains the new information
- Three new elements: `test_authorized_for_otc`, `test_authorized_for_home`, and `test_authorized_for_unproctored`
- Schema updated to include the new elements
- No code changes as the existing mappers and filters are used. 
- New unit-tests. 

### Testing
- [x] Tested locally?
- [x] Ran `quickTest all`?
- [x] Ran `./prime test` against local Docker ReportStream container?
- [x] Downloaded a file from `http://localhost:7071/api/download`?
- [x] Added tests?

### Security
- [ ] Did you check for sensitive data, and remove any? 
- [ ] Does logging contain sensitive data?  
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

### Process
- [x] Includes a summary of what a code reviewer should verify?
- [x] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

## Security
*List potential security threats and mitigations in the PR*

## Fixes
- #1225 

## To Be Done
*Create GitHub issues to track the work remaining, if any*

### CODE REVIEW

File: prime-router/src/test/resources/test_data_files/Hl7_ORU-R01/CE-20200415-0001.internal.csv [hunk 1]
```diff
@@ -1,2 +1,2 @@
...
...ge_id,previous_message_id,message_profile_id,reason_for_study,receiving_application,receiving_facility,reference_range,reporting_facility,reporting_facility_name,reporting_facility_clia,resident_congregate_setting,result_format,sending_application,specimen_collection_date_time,specimen_collection_method,specimen_collection_site,specimen_description,specimen_id,specimen_role,specimen_source_site_code,specimen_type,symptomatic_for_disease,test_authorized_for_otc,test_authorized_for_home,test_authorized_for_unproctored,test_kit_name_id,test_kit_name_id_type,test_kit_name_id_cwe_version,test_method_description,test_performed_code,test_performed_name,test_performed_system_abbr,test_performed_system,test_performed_system_version,test_result,test_result_date,test_result_report_date,test_result_status,test_result_sub_id,test_result_units,testing_lab_accession_number,testing_lab_city,testing_lab_county,testing_lab_county_code,testing_lab_country,testing_lab_id,testing_lab_id_assigner,testing_lab_clia,testing_lab_name,testing_lab_specimen_id,testing_lab_specimen_received_datetime,testing_lab_state,testing_lab_street,testing_lab_street2,testing_lab_zip_code,testing_lab_phone_number,value_type
```
Comment (RickHawesUSDS) [inline prime-router/src/test/resources/test_data_files/Hl7_ORU-R01/CE-20200415-0001.internal.csv:1]: These test files needed to have the new columns added to them.

File: prime-router/src/test/kotlin/MapperTests.kt [hunk 2]
```diff
@@ -119,6 +119,46 @@ class MapperTests {
     }
 
 
+    @Test
+    fun `test livdLookup supplemental table by device_id`() {
+        val lookupTable = LookupTable.read("./metadata/tables/LIVD-Supplemental-2021-06-07.csv")
```
Comment (RickHawesUSDS) [inline prime-router/src/test/kotlin/MapperTests.kt:124]: These tests prove that the livdLookup mapper work on this supplemental table.

File: prime-router/src/main/kotlin/FakeReport.kt [hunk 3]
```diff
@@ -130,6 +130,10 @@ class FakeDataService {
                                 "to ${element.tableColumn}"
                         )
                 }
+                element.table?.startsWith("LIVD-Supplemental") == true -> {
+                    if (element.tableColumn == null) return ""
+                    element.default ?: ""
+                }
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/FakeReport.kt:136]: Always generate a default value for the supplemental table.

**Review by jimduff-usds — APPROVED** [review]
LGTM

**Pull request merged**